<launch>
    <arg name="imu_div_name"  default="/dev/ttyUSB0"/>
	<arg name="imu_topic"  default="imu_raw"/>
	<arg name="mag_topic"  default="mag_raw"/>
	<arg name="imu_frame"  default="forsense"/>
	<arg name="sample_rate" default="100" />
    <arg name="ns" default="robot1"/>
	<!--"frame_id" represents the id of the point cloud data published to ROS-->
    <group ns="$(arg ns)" >
    <node pkg="forsense_imu" name="forsense" type="forsense_imu"  output="screen">
		<param name="imu_div_name"  type="string" value="$(arg imu_div_name)"/>
		<param name="imu_topic"  type="string" value="$(arg imu_topic)"/>
        <param name="mag_topic"  type="string" value="$(arg mag_topic)"/>
		<param name="imu_frame"  type="string" value="$(arg imu_frame)"/>
        <param name="sample_rate"  type="int" value="$(arg sample_rate)" />
        <param name="ns"  type="string" value="$(arg ns) "/>
    </node>
    </group>

</launch>

